ci/capture_android_release.sh

```diff
@@ -64,6 +64,7 @@ function release_capture_sdk() {
       "$sdk_repo/ci/NOTICE.txt" \
       "$name.pom" \
       "$name-javadoc.jar" \
+      "$name-sources.jar" \      
```

Comment:
@Augustyniak I assume this is all I need to do since by looking at the logs it seems we do download and unzip the sources from gh here